tests/late_joiners.py

```diff
@@ -52,18 +52,19 @@ def get_node_local_commit(node):
         return r.commit, r.global_commit
```

Comment:
When run with PBFT, the swallowing of the RuntimeError when transactions are not found by the status call should be removed. In the Raft case this can happen, but with PBFT, as per discussion with @ashamis, transactions that are assigned transaction ids should never be lost, even in rollbacks.

Reply:
Sorry, I see this has been done already, hadn't seen it.